### PR TITLE
Inspector settings docs

### PR DESCRIPTION
**Pull request by CoderDake**
_Fixes:_ https://github.com/flutter/website/issues/9420

_Relates to:_ https://github.com/flutter/website/pull/9853
## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:520]:
```suggestion
## Inspector settings
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:523]:
```suggestion
### Enable hover inspection
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:527]:
```suggestion
Toggling this value enables or disables the hover inspection functionality.
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:529]:
```suggestion
### Display package directories
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:531]:
```suggestion
By default, DevTools limits the widgets displayed in the widget tree
to those from the project's root directory, and those from Flutter. 
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:533]:
```suggestion
To display other widgets, add the parent directory of those widgets to the package directories.
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:535]:
```suggestion
For example, consider the following directory structure:
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:545]:
```suggestion
Running your app from `project_foo_app` displays only widgets from
`project_foo/pkgs/project_foo_app` in the widget inspector.
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:548]:
```suggestion
To show widgets from `widgets_A` in the widget tree,
add `project_foo/pkgs/widgets_A` to the package directories.
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:551]:
```suggestion
To display _all_ widgets from your project root in the widget tree,
add `project_foo` to the package directories.
```

File: src/tools/devtools/inspector.md
Comment (sfshaza2) [inline src/tools/devtools/inspector.md:554]:
```suggestion
Changes to your package directories persist the next time the
widget inspector is opened for the app.
```

**Review by sfshaza2 — APPROVED** [review]
Thanks, @CoderDake, for this update! lgtm with some wordsmithing. I'm approving to make it easier to land, but please first make the grammatical edits.

File: src/tools/devtools/inspector.md
Comment (kenzieschmoll) [reply src/tools/devtools/inspector.md:545]: "in the widget inspector summary tree"

File: src/tools/devtools/inspector.md
Comment (kenzieschmoll) [reply src/tools/devtools/inspector.md:531]: I would add a sentence after this: "This filtering only applies to the widgets in the Inspector Widget Tree (left side of the Inspector) -- not the Widget Details Tree (right side of the Inspector in the same tab view as the Layout Explorer). In the Widget Details Tree, you will be able to see all widgets in the tree from all packages."

**Review by domesticmouse — APPROVED** [review]
LGTM once @sfshaza2's edits are landed

**Comment by CoderDake** [comment]
Thank you for the help!

**Pull request merged**
